packages/lodestar/src/db/api/beacon/repository.ts

```diff
@@ -65,7 +65,7 @@ export abstract class BulkRepository<T> extends Repository<T> {
 
   public async getAllBetween(lowerLimit: number, upperLimit: number): Promise<T[]> {
     const data = await this.db.search({
-      gt: encodeKey(this.bucket, lowerLimit),
+      gt: encodeKey(this.bucket, lowerLimit || Buffer.alloc(0)),
```

Comment:
should be just 0 not buffer alloc

Reply:
With that it didn't return the 0th index element for some reason. I needed to add this to have it be included. There wasn't a good way to set up a unit test but if you make that change and check the values returned in the e2e test, the 0th index was never included.